x-transactions/x-transaction.js

```diff
@@ -7,10 +7,10 @@ export default class XTransaction extends MixinRedux(XElement) {
     html() {
         return `
             <div class="timestamp" title="">pending...</div>
-            <x-identicon sender></x-identicon>
+            <div class="identicon"><x-identicon sender></x-identicon></div>
```

Comment:
Why not add
```js
styles() {
    return [ ...super.styles(), 'identicon' ];
}
```
to `x-identicon`?
Wouldn't that be the clean approach?
or is it really a parent element we need to apply the style to?

Reply:
I think it is. Not 100% sure now anymore.